components/switch/index.css

```diff
@@ -56,6 +56,9 @@
 	--spectrum-switch-handle-size: var(--spectrum-switch-handle-size-medium);
 	--spectrum-switch-handle-selected-size: var(--spectrum-switch-handle-selected-size-medium);
 
+	--spectrum-switch-animation-duration-100: var(--spectrum-animation-duration-100);
+	--spectrum-switch-animation-duration-200: var(--spectrum-animation-duration-200);
```

Comment:
From the switch migration, I noticed these 2 custom props were missing. I don't think we should see any difference, expect that the styles have one extra layer of token abstraction.